examples/peg_pickup_demo.py

```diff
@@ -4,100 +4,109 @@
 # at an a-priori known location. Runs in simulation.
 #
 ##
-
+import ipdb
```

Comment:
Oof. Just realized that I left some debugging scripts in here. Will remove these now...